pyproject.toml

```diff
@@ -1,19 +1,21 @@
-[tool.poetry]
-name = "todo-app"
-version = "0.1.0"
-description = "A simple ToDo app for your DevOps apprenticeship"
-authors = []
-
-[tool.poetry.dependencies]
-python = "^3.7"
-Flask = "^2.0.3"
-python-dotenv = "^0.14.0"
-requests = "^2.28.1"
-gunicorn = "^20.1.0"
-
-[tool.poetry.dev-dependencies]
-pytest = "^7.1.3"
-
-[build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+[tool.poetry]
+name = "todo-app"
+version = "0.1.0"
+description = "A simple ToDo app for your DevOps apprenticeship"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.7"
+Flask = "^2.0.3"
+python-dotenv = "^0.14.0"
+requests = "^2.28.1"
+gunicorn = "^20.1.0"
+pymongo = "^4.3.3"
+mongomock = "^4.1.2"

```

Comment:
Obviously minor, but moving this to the dev-dependencies would slightly reduce the attack surface in our deployed app